tests/ImageSharp.Tests/Primitives/ColorMatrixTests.cs

```diff
@@ -75,10 +76,10 @@ public void ColorMatrixMultiply()
         m.M44 = (value1.M41 * value2.M14) + (value1.M42 * value2.M24) + (value1.M43 * value2.M34) + (value1.M44 * value2.M44);
 
         // Fifth row
-        m.M51 = (value1.M51 * value2.M11) + (value1.M52 * value2.M21) + (value1.M53 * value2.M31) + (value1.M54 * value2.M41) + value2.M51;
-        m.M52 = (value1.M51 * value2.M12) + (value1.M52 * value2.M22) + (value1.M53 * value2.M32) + (value1.M54 * value2.M52) + value2.M52;
-        m.M53 = (value1.M51 * value2.M13) + (value1.M52 * value2.M23) + (value1.M53 * value2.M33) + (value1.M54 * value2.M53) + value2.M53;
-        m.M54 = (value1.M51 * value2.M14) + (value1.M52 * value2.M24) + (value1.M53 * value2.M34) + (value1.M54 * value2.M54) + value2.M54;
+        m.M51 = (value1.M51 * value2.M11) + (value1.M52 * value2.M21) + (value1.M53 * value2.M31) + (value1.M54 * value2.M41);
```

Comment:
What am I missing here. Why did this change?

Reply:
Probably because I did something wrong and misunderstood the logic ;) 
Gonna check it.

Reply:
Use `Matrix3x2` as a reference. That's what I did the first time round 👍

Reply:
I think I just forget adding the values of the 5th row

Reply:
I think I fixed it

Reply:
Yep. Seems to be working. I just pushed an additional fix to optimize the transformation. You were adding each element separately when it could be a single instruction. 

I'd love to see a benchmark, if possible, compared to main? It's just about midnight here so I won't be able to do it.

Reply:
Gonna have a try setting up a benchmark. Do you have a hint where I should start?

Reply:
There’s a benchmark project in the solution. I’d copy bits from the PremultiplyVector4 benchmark into a new file and directly test ColorNumerics.

Reply:
This is a Benchmark over the Multiply code which is in the unit tests

``` ini

BenchmarkDotNet=v0.13.0, OS=macOS 13.2.1 (22D68) [Darwin 22.3.0]
Intel Core i9-9880H CPU 2.30GHz, 1 CPU, 16 logical and 8 physical cores
.NET SDK=7.0.200
  [Host]     : .NET 6.0.14 (6.0.1423.7309), X64 RyuJIT
  Job-HOQWTB : .NET 6.0.14 (6.0.1423.7309), X64 RyuJIT

Runtime=.NET 6.0  Arguments=/p:DebugType=portable  

```

Before:

|         Method |     Mean |    Error |   StdDev | Gen 0 | Gen 1 | Gen 2 | Allocated |
|--------------- |---------:|---------:|---------:|------:|------:|------:|----------:|
|      Transform | 34.36 ns | 0.370 ns | 0.346 ns |     - |     - |     - |         - |
| Transform_Span | 61.50 ns | 1.248 ns | 3.374 ns |     - |     - |     - |         - |

After:

|         Method |     Mean |    Error |   StdDev | Gen 0 | Gen 1 | Gen 2 | Allocated |
|--------------- |---------:|---------:|---------:|------:|------:|------:|----------:|
|      Transform | 30.11 ns | 0.215 ns | 0.180 ns |     - |     - |     - |         - |
| Transform_Span | 44.29 ns | 0.787 ns | 0.737 ns |     - |     - |     - |         - |